ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py

```diff
@@ -217,6 +217,40 @@ def paddle_unfold_handler(draw, dtype):
 # ------------ #
 
 
+# test function for bilinear
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.bilinear",
+    dtype_x1_x2_weight_bias=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid", full=False),
+    ),
+)
+def test_paddle_bilinear(
+    *,
+    dtype_x1_x2_weight_bias,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    dtype, x1, x2, weight, bias = dtype_x1_x2_weight_bias
```

Comment:
dtype_and_values helper function return max 3 values,
please check the definition of it and use the appropriate helper function for generating the test data.

Reply:
![image](https://github.com/unifyai/ivy/assets/22115102/4d64f690-49b0-4722-b448-deb330a17760)